code/modules/clothing/under/accessories/armor.dm

```diff
@@ -226,3 +290,50 @@
 	name = "\improper AB- blood patch"
 	desc = "An embroidered patch indicating the wearer's blood type as AB NEGATIVE."
 	icon_state = "abnegtag"
+
+/////////////////
+// Helmet Covers
+/////////////////
+
+obj/item/clothing/accessory/armor/helmcover
+	name = "helmet cover"
+	desc = "A fabric cover for armored helmets."
+	icon_override = 'icons/mob/ties.dmi'
+	icon = 'icons/obj/clothing/modular_armor.dmi'
+	icon_state = "helmcover_blue"
+	slot = ACCESSORY_SLOT_HELM_C
+
+/obj/item/clothing/accessory/armor/helmcover/blue
+	name = "blue helmet cover"
+	desc = "A fabric cover for armored helmets in a bright blue color."
+	icon_state = "helmcover_blue"
+
+/obj/item/clothing/accessory/armor/helmcover/navy
+	name = "navy blue helmet cover"
+	desc = "A fabric cover for armored helmets. This one is colored navy blue."
+	icon_state = "helmcover_navy"
+
+/obj/item/clothing/accessory/armor/helmcover/green
+	name = "green helmet cover"
+	desc = "A fabric cover for armored helmets. This one has a woodland camouflage pattern."
+	icon_state = "helmcover_green"
+
+/obj/item/clothing/accessory/armor/helmcover/tan
+	name = "tan helmet cover"
+	desc = "A fabric cover for armored helmets. This one has a desert camouflage pattern."
+	icon_state = "helmcover_tan"
+
+/obj/item/clothing/accessory/armor/helmcover/nt
+	name = "\improper NanoTrasen helmet cover"
+	desc = "A fabric cover for armored helmets. This one has NanoTrasen's colors."
+	icon_state = "helmcover_nt"
+
+/obj/item/clothing/accessory/armor/helmcover/pcrc
+	name = "\improper PCRC helmet cover"
+	desc = "A fabric cover for armored helmets. This one is colored navy blue and has a tag in the back with the words PROXIMA CENTAURI RISK CONTROL printed in cyan lettering on it."
+	icon_state = "helmcover_pcrc"
+
+/obj/item/clothing/accessory/armor/helmcover/saare
+	name = "\improper SAARE helmet cover"
+	desc = "A fabric cover for armored helmets. This one has SAARE's colors."
```

Comment:
Just checking; what ARE those colors, here?

Reply:
Literally all of these were just copy+pasted over. They're not going to be obtainable in game.